drivers/sensor/aht20/aht20.h

```diff
@@ -0,0 +1,48 @@
+/*
```

Comment:
I'm not sure this header is really needed at all? All of this could be privately placed in the .c if you like.